td/tracking/models.py

```diff
@@ -65,6 +70,17 @@ def __unicode__(self):
     __unicode__.allow_tags = True
     __unicode__.admin_order_field = "language"
 
+    @property
+    def lang_id(self):
+        return self.language.id
+
+    @classmethod
+    def lang_data(self):
```

Comment:
Since it refers to class attributes instead of the instance, we should use `cls` instead of `self` when using the `@classmethod` decorator.